cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DocumentingTestBase.scala

```diff
@@ -572,7 +573,10 @@ abstract class DocumentingTestBase extends JUnitSuite with DocumentationHelper w
   protected def databaseConfig(): util.Map[Setting[_], Object] =
     Map[Setting[_], Object](
       OnlineBackupSettings.online_backup_listen_address -> new SocketAddress("127.0.0.1", 0),
-      OnlineBackupSettings.online_backup_enabled -> java.lang.Boolean.FALSE
+      OnlineBackupSettings.online_backup_enabled -> java.lang.Boolean.FALSE,
+      GraphDatabaseInternalSettings.text_indexes_enabled -> java.lang.Boolean.TRUE,
+      GraphDatabaseInternalSettings.range_indexes_enabled -> java.lang.Boolean.TRUE,
+      GraphDatabaseInternalSettings.point_indexes_enabled -> java.lang.Boolean.TRUE
```

Comment:
I have a PR enabling the feature and removing these settings (that is waiting on drop discussions), so whoever is last to merge needs to remove the usage of these settings here too